packages/vite/rollup.config.ts

```diff
@@ -44,6 +44,7 @@ const clientConfig = defineConfig({
 const sharedNodeOptions = defineConfig({
   treeshake: {
     moduleSideEffects(id, external) {
+      id = id.replaceAll('\\', '/')
```

Comment:
On windows, `\\` is used and the following `if` block was not used.

Reply:
Interesting. I thought Rollup already normalizes it for us.

Reply:
Actually it's Vite's feature 🙂
https://vite.dev/guide/api-plugin.html#path-normalization:~:text=On%20the%20other%20hand%2C%20Rollup%20keeps%20resolved%20paths%20untouched%20by%20default